.eslintignore

```diff
@@ -0,0 +1 @@
+messaging/react-typescript-webpack/
```

Comment:
does this not lint? :stuck_out_tongue:

Reply:
It's complaining about Webpack configs, since they're JS and not TS. Complains about using `const` because the current config only understands ES5. I could add a config file that makes ESLint understand Node JavaScript, but I'd rather not have a random ESLint config in the example directory.